### PR TITLE
Rerender

### PR DESCRIPTION
**Pull request by djsutherland**
Adds a build with VC 9, which wasn't in the previous rererending: https://ci.appveyor.com/project/conda-forge/cyflann-feedstock/build/1.0.3/job/jgpvds5999q4g8we#L183.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by djsutherland** [comment]
Okay, VC 9 build doesn't work. It doesn't like this block: 

https://github.com/mariusmuja/flann/blob/master/src/cpp/flann/util/serialization.h#L109-L115

Probably could just patch to kill that.

**Comment by jakirkham** [comment]
Patching it out sounds good. Seems like they are just working around a VS 2013 bug. As we don't use that version of VS (and likely never will), patching it out seems fine. Upstream might prefer version bounds for that though.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,19 +12,23 @@ source:
...
+        - gcc     # [osx]
         - cmake
+    host:
+        - libgcc  # [osx]
+        - hdf5
     run:
-        - libgcc  # [unix]
-        - hdf5 1.10.1|1.10.1.*
+        - libgcc  # [osx]
+        - {{ pin_compatible('hdf5', max_pin='x.x.x') }}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Don't think we need the `pin_compatible` as that should happen automatically now thanks to PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/46 ), but will let @isuruf clarify.

Comment (djsutherland) [reply recipe/meta.yaml]: Hmm. So, with `pin_compatible`, `conda render recipe` gives me:
```
    run:
        - hdf5 >=1.10.1,<1.10.2.0a0
        - libgcc
```
but without it, I get
```
    run:
        - hdf5
        - libgcc
```
I'm not totally sure about how `conda render` interacts with this stuff, but I thought that was bad. @isuruf / @msarahan?

Comment (isuruf) [reply recipe/meta.yaml]: Do `conda render recipe -m /path/to/conda_build_config.yaml`

Comment (djsutherland) [reply recipe/meta.yaml]: Aha! Thanks @isuruf. Will remove `pin_compatible`.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by djsutherland** [comment]
Hmm; so I re-rendered off of 79980f3 and it said up to date. Guess something happened to change the rendering either in conda 4.5.1 -> 4.5.2 or conda-build 3.8.1 -> 3.9.2; maybe conda-smithy should warn when conda or conda-build are out of date?

**Comment by jakirkham** [comment]
Yeah, so `conda-build` fixed issue ( https://github.com/conda/conda-build/issues/2701 ) at some point. While it did change the result, we aren't aware of any significant problems caused by that bug.

That sounds like a good idea. Could you please raise an issue over at `conda-smithy`?

**Comment by jakirkham** [comment]
Thanks @dougalsutherland :)

**Comment by jakirkham** [comment]
FWIW the metadata on Anaconda.org also matches what one would expect. In particular, here is a snippet from `linux-64/flann-1.9.1-h1e41055_2.tar.bz2`.

```
depends | hdf5 >=1.10.1,<1.10.2.0a0, libstdcxx-ng >=4.9
```


ref: https://anaconda.org/conda-forge/flann/files/modal/info/5ae8fa5696e38211da40bac8

**Pull request merged**